lib/src/instances/option.dart

```diff
@@ -84,11 +228,103 @@ class Some<T> extends Option<T> {
   }
 
   @override
+  bool contains(Object element) => element == value;
+
+  @override
+  T elementAt(int index) {
+    if (index != 0) {
+      throw new RangeError.index(index, this, "index", null, 1);
+    }
+
+    return value;
+  }
+
+  @override
+  bool every(bool f(T element)) => f(value);
+
+  @override
+  IterableMonad<S> expand<S>(Iterable<S> f(T element)) =>
```

Comment:
Reviewed offline and couldn't figure out how to do this since the API required by `Iterable` is not flexible at all. We agreed on annotating `Iterable.expand` as `@deprecated` favoring `flatMap` instead.